management_service/app/routers/orders.router.ts

```diff
@@ -4,78 +4,73 @@ import { Router, Request, Response } from "express";
 const ordersRouter = Router() as NamedRouter;
 ordersRouter.prefix = "orders";
 
-ordersRouter.get("/:id", (res: Response) => {
-    try {
-        // This is a placeholder for the actual order retrieval logic
+ordersRouter.get("/:id", (req: Request, res: Response) => {
```

Comment:
I don't see req being used in the method. Either leave it out _or_ prefix it with an underscore so it passes the linter.

`_req:Request`